dotCMS/src/integration-test/java/com/dotmarketing/portlets/containers/business/FileAssetContainerUtilTest.java

```diff
@@ -116,6 +135,70 @@ public void test_getHost_demo_default_demo() throws Exception {
         }
     }
 
+    /**
+     * Method to test: {@link FileAssetContainerUtil#fromAssets(Host, Folder, List, boolean, boolean)}
+     * Given Scenario: Creates a container with a default layout
+     * ExpectedResult: Expected result that the result should contains the default container layout
+     * @throws Exception
+     */
+    @Test
+    public void test_fromAssets_default_layout() throws Exception {
+
+        //Getting the current default host
+        final Host currentDefaultHost     = APILocator.getHostAPI().findDefaultHost(APILocator.systemUser(), false);
+        final Folder containerFolder      = this.createAppContainerTest(currentDefaultHost);
+        final String contentTypeVariable  = "testfa"+System.currentTimeMillis();
+        ContentType fileAssetContentType  = ImmutableFileAssetContentType.builder()
+                .name("FileAssetTestContentType").variable(contentTypeVariable).build();
+        fileAssetContentType              = contentTypeAPI.save(fileAssetContentType);
+        final List<FileAsset> assets      = Arrays.asList(
+                this.createFileAsset("container", ".vtl",
+                        "$dotJSON.put(\"title\", \"Test VTL and Valid and Invalid Content Types\")\n" +
+                        "$dotJSON.put(\"max_contentlets\", 25)\n" +
+                        "$dotJSON.put(\"useDefaultLayout\",\"*\")", containerFolder, fileAssetContentType),
+                this.createFileAsset("default_container", ".vtl", "$title", containerFolder, fileAssetContentType));
+        final boolean showLive            = false;
+        final boolean includeHostOnPath   = false;
+
+        final Container container = FileAssetContainerUtil.getInstance().fromAssets(currentDefaultHost,
+                containerFolder, assets, showLive, includeHostOnPath);
+
+        Assert.assertNotNull(container);
+        Assert.assertTrue(container instanceof FileAssetContainer);
+        Assert.assertEquals(1   , FileAssetContainer.class.cast(container).getContainerStructuresAssets().size());
+        Assert.assertNotNull(FileAssetContainer.class.cast(container).getDefaultContainerLayoutAsset());
+        Assert.assertEquals("default_container.vtl",FileAssetContainer.class.cast(container).getDefaultContainerLayoutAsset().getFileName());
+    }
+
+    private FileAsset createFileAsset (final String fileName1,
+                                       final String fileExtension,
+                                       final String fileContent,
+                                       final Folder root1,
+                                       final ContentType fileAssetContentType) throws Exception {
```

Comment:
Codacy found an issue: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6347242)